sbndaq-artdaq/Generators/Common/CAENV1730Readout_generator.cc

```diff
@@ -679,6 +685,49 @@ void sbndaq::CAENV1730Readout::Write_ADC_CalParams_V1730(int handle, int ch, uin
   }
 }
 
+void sbndaq::CAENV1730Readout::ConfigureClkToTrgOut()
+{
+  /* Check to output ONLY CLK OR CLK PHASE */
+  if ( (fOutputClk == 1) & (fOutputClkPhase == 1) ){
```

Comment:
Very minor, just a preference of mine :)
```suggestion
  if ( fOutputClk && fOutputClkPhase ){
```